tests/unit/test_changelog.py

```diff
@@ -39,8 +45,12 @@
         ),
     ],
 )
-def test_entry_evr(header, evr):
-    assert ChangelogEntry(header, [""]).evr == evr
+def test_entry_evr(header, evr_str):
+    evr = ChangelogEntry(header, [""]).evr
+    if evr_str:
+        assert evr == EVR.from_string(evr_str)
+    else:
+        assert evr is None
```

Comment:
Wouldn't it make more sense to parametrize the test with `EVR` objects (or `None`), like e.g. here?

https://github.com/packit/specfile/blob/c0a98a85d9e43f59447248bb7f5258c2406ce9f1/tests/unit/test_utils.py#L91-L100

It's more verbose and a bit more work, but I like it more that way :slightly_smiling_face: